src/main/java/ru/vk/itmo/shishiginstepan/PersistentStorage.java

```diff
@@ -0,0 +1,39 @@
+package ru.vk.itmo.shishiginstepan;
+
+import ru.vk.itmo.Entry;
+
+import java.lang.foreign.MemorySegment;
+import java.nio.file.Path;
+import java.util.Collection;
+
+public class PersistentStorage {
+    // TODO заменить на интерфейс сстейбл
+    private final SimpleSSTable sstable;
+
+    PersistentStorage(Path basePath) {
+        this.sstable = new SimpleSSTable(basePath);
+    }
+
+    public void store(Collection<Entry<MemorySegment>> data) {
+        long dataSize = 0;
+        for (var entry: data) {
+            dataSize += entry.value().byteSize() + entry.key().byteSize() + 16;
```

Comment:
Так было бы чуть понятнее:
```suggestion
            dataSize += Long.BYTES + entry.value().byteSize() + Long.BYTES + entry.key().byteSize();
```